icu4c/source/test/intltest/measfmttest.cpp

```diff
@@ -5066,6 +5068,30 @@ void MeasureFormatTest::TestInternalMeasureUnitImpl() {
                  std::move(m2m).build(status).getIdentifier());
 }
 
+void MeasureFormatTest::TestMilligramPerDeciliter() {
+    IcuTestErrorCode status(*this, "TestMilligramPerDeciliter");
+    auto milligram_per_deciliter = MeasureUnit::forIdentifier("milligram-per-deciliter", status);
+    auto milligram_per_deciliter2 = MeasureUnit::getMilligramPerDeciliter();
+    auto milligram_per_deciliter3 = MeasureUnit::createMilligramPerDeciliter(status);
+
+    assertSuccess("test", status);
+    assertEquals("test", milligram_per_deciliter.getIdentifier(), "milligram-per-deciliter");
+    assertEquals("test", milligram_per_deciliter2.getIdentifier(), "milligram-per-deciliter");
+    assertEquals("test", milligram_per_deciliter3->getIdentifier(), "milligram-per-deciliter");
+
+    auto milligram_per_deciliter_new =
+        MeasureUnit::forIdentifier("milligram-ofglucose-per-deciliter", status);
+    auto milligram_per_deciliter_new2 = MeasureUnit::getMilligramOfglucosePerDeciliter();
+    auto milligram_per_deciliter_new3 = MeasureUnit::createMilligramOfglucosePerDeciliter(status);
+
+    assertSuccess("test", status);
+    assertEquals("test", milligram_per_deciliter_new.getIdentifier(),
+                 "milligram-ofglucose-per-deciliter");
+    assertEquals("test", milligram_per_deciliter_new2.getIdentifier(),
+                 "milligram-ofglucose-per-deciliter");
+    assertEquals("test", milligram_per_deciliter_new3->getIdentifier(),
+                 "milligram-ofglucose-per-deciliter");
+}
 
```

Comment:
These tests do not test the problem described in the bug. That problem was about errors when *formatting* using e.g. milligram_per_deciliter or milligram_per_deciliter2 or milligram_per_deciliter3 as created above. It was not about problems in creating or equating the MeasureUnits themselves.

Reply:
Acknowledged.